### PR TITLE
Swap Llava export arg order

### PR DESCRIPTION
**Pull request by jackzhxng**
### Summary
Swaps Llava export arg order, such that forward takes `embeddings, cache_position`, instead of `cache_position, embeddings`.

### Test plan
Existing Llava tests

### CODE REVIEW

**Review by kirklandsign — APPROVED** [review]

File: examples/models/llava/model.py [hunk 1]
```diff
@@ -405,5 +405,5 @@ def _get_image_dynamic_shapes(self):
 
     def _get_prompt_dynamic_shapes(self):
         dim = torch.export.Dim("token_dim", min=2, max=self.max_seq_len)
-        text_model_dynamic_shapes = ({0: 1}, {1: dim})
+        text_model_dynamic_shapes = ({1: dim}, {0: 1})
```
Comment (JacobSzwejbka) [inline examples/models/llava/model.py:408]: nit: 1 and 0 are the dims of a single tensor. So swapping doesnt do anything

Comment (JacobSzwejbka) [reply examples/models/llava/model.py:408]: also do you even have to specify that the batch is static 1?

Comment (jackzhxng) [reply examples/models/llava/model.py:408]: {0:1} is the dynamic shapes for the first arg, {1:dim} is for the second arg, so I swap them here
